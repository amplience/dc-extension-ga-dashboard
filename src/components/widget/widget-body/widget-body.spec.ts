import { render } from '@testing-library/svelte';
import WidgetBody from './widget-body.svelte';

describe('WidgetBody', () => {
  it('should render the Widget component', () => {
    const { container } = render(WidgetBody, {});

    expect(container.firstChild).toMatchSnapshot();
  });
});
