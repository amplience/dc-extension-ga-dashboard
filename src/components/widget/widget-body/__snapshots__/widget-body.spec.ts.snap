// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WidgetBody should render the Widget component 1`] = `
<div>
  <section
    class="widget-body svelte-4p1s5s"
  />
</div>
`;
